src/js/modules/ExternalLink.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ExternalLinkSVG from 'js/svg/ExternalLink'
+
+const ExternalLink = (props) => (
+  <a href={props.to} className="coa-ExternalLink">
```

Comment:
should this link open in a new tab? the little icon makes me think it should, like the footer links, but I'm not sure.

Reply:
yup, great call.